tutorials/preprocessing/90_eyetracking_data.py

```diff
@@ -98,126 +96,108 @@
-raw.plot(events=events, event_id={"Flash": 3}, event_color="g")
+# .. important:: By default, :func:`~mne.preprocessing.eyetracking.interpolate_blinks`,
+#           will only interpolate blinks in pupil channels. Passing
+#           ``interpolate_gaze=True`` will also interpolate the blink periods of the
+#           eyegaze channels. Be aware, however, that eye movements can occur
+#           during blinks which makes the gaze data less suitable for interpolation.
 
 # %%
-# :func:`~mne.preprocessing.eyetracking.interpolate_blinks` performs a simple linear
-# interpolation of the pupil size data during blink periods. the ``buffer`` keyword
-# argument specifies the amount of time (in seconds) before and after the blinks to
-# include in the interpolation. This is helpful because the ``blink`` annotations
-# do not always capture the entire blink in the signal. We specified a value of ``.05``
-# seconds (50 ms), which is slightly more than the default value of ``.025``.
+# Extract common stimulus events from the data
+# --------------------------------------------
+#
+# In this experiment, a photodiode attached to the display screen was connected to both
+# the EEG and eye-tracking systems. The photodiode was triggered by the the light flash
+# stimuli, causing a signal to be sent to both systems simultaneously, signifying the
+# onset of the flash. The photodiode signal was recorded as a digital input channel in
+# the EEG and eye-tracking data. MNE loads these data as a :term:`stim channel`.
+#
+# We'll extract the flash event onsets from both the EEG and eye-tracking data, as they
+# are necessary for aligning the data from the two recordings.
 
+et_events = mne.find_events(raw_et, min_duration=0.01, shortest_event=1, uint_cast=True)
+event_dict = {"Flash": 2}
+eeg_events = mne.find_events(raw_eeg, stim_channel="DIN3")
 # %%
-# Dealing with high frequency noise
-# ---------------------------------
-# From the plot above, we notice that there is some high frequency noise in the pupil
-# signal. We can remove this noise by low-pass filtering the data:
+# Align the eye-tracking data with EEG the data
+# ---------------------------------------------
+#
+# In this dataset, eye-tracking and EEG data were recorded simultaneously, but on
+# different systems, so we'll need to align the data before we can analyze them
+# together. We can do this using the :func:`~mne.preprocessing.realign_raw` function,
+# which will align the data based on the timing of the shared events that are present in
+# both :class:`~mne.io.Raw` objects. We'll use the shared photodiode events we extracted
+# above, but first we need to convert the event onsets from samples to seconds. Once the
+# data have been aligned, we'll add the EEG channels to the eye-tracking raw object.
+
+# Convert event onsets from samples to seconds
+et_din_times = et_events[:, 0] / raw_et.info["sfreq"]
+eeg_din_times = eeg_events[:, 0] / raw_eeg.info["sfreq"]
```

Comment:
Clearer variable name, "DIN" is never explained

```suggestion
et_flash_times = et_events[:, 0] / raw_et.info["sfreq"]
eeg_flash_times = eeg_events[:, 0] / raw_eeg.info["sfreq"]
```